### PR TITLE
fix: [PL-48492]: removed dependency of @harness/help-panel from uicore

### PR DESCRIPTION
**Pull request by SahilKr24**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/uicore/package.json [hunk 1]
```diff
@@ -18,9 +18,8 @@
     "@blueprintjs/core": "^3.20.0",
     "@blueprintjs/datetime": "3.13.0",
     "@blueprintjs/select": "3.10.0",
-    "@harness/help-panel": "^1.14.0",
     "@harnessio/design-system": "1.6.0-beta.2",
-    "@harnessio/icons": "^2.0.0-beta.2",
+    "@harnessio/icons": "^2.0.0-beta.8",
```
[the remark this answers is not in the document]
Comment (SahilKr24) [reply packages/uicore/package.json:22]: I tried but it wouldn't update with yarn update -D "@harnessio/icons"@"^2.0.0-beta.8".
Let me try from the other directory

[the remark this answers is not in the document]
Comment (SahilKr24) [reply packages/uicore/package.json:22]: Still not working, not use why. There is only one yarn lock file at the root of the repo and it doesn't have "@harnessio/icons" anywhere. 😭

[the remark this answers is not in the document]
Comment (SahilKr24) [reply packages/uicore/package.json]: I reverted the upgrade, it seems to work fine without the version bump.

[the remark this answers is not in the document]
Comment (tan-nhu) [reply packages/uicore/package.json]: Push the new `yarn.lock`? I still don't see it here

[the remark this answers is not in the document]
Comment (SahilKr24) [reply packages/uicore/package.json]: The `yarn.lock` file doesn't update when I run the yarn commands. Check attached screenshot.
image

**Review by tan-nhu — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**